dali/python/backend_impl.cc

```diff
@@ -1288,13 +1288,13 @@ PYBIND11_MODULE(backend_impl, m) {
 
   m.def("GetCxx11AbiFlag", &GetCxx11AbiFlag);
 
-  m.def("HasCudaContext", []{
-    if (!cuInitChecked()) {
+  m.def("IsDriverInitialized", []{
+    int place_holder = 0;
+    // driver is not initialized so we don't have context
+    if (CUDA_SUCCESS != cuDeviceGetCount(&place_holder)) {
```

Comment:
Are we fine with loading libcuda (if it's there)? What would happen if it isn't?